x-pack/plugins/synthetics/e2e/helpers/record_video.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import fs from 'fs';
+import Runner from '@elastic/synthetics/src/core/runner';
+import { after, Page } from '@elastic/synthetics';
+
+const SYNTHETICS_RUNNER = Symbol.for('SYNTHETICS_RUNNER');
+
+// @ts-ignore
+export const runner: Runner = global[SYNTHETICS_RUNNER];
+
+export const recordVideo = (page: Page) => {
```

Comment:
Is this custom helper purely for the purpose of naming the video?

Reply:
yes